src/Compiler/Microsoft.AspNetCore.Razor.Language/src/RazorCodeDocumentExtensions.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
File encoding change?

Reply:
I haven't merged in main, due to local changes I'm not committing that allow me to get the "before" state from a single benchmark run, so I assume this isn't real, but I can merge/rebase and confirm before I merge the PR.

Reply:
Had to manually fix this 🤷‍♂️